multimedia/mpv/Portfile

```diff
@@ -304,24 +302,6 @@ foreach ver ${python.versions} {
     eval $variant_line
 }
 
-# Create legacy python variants, mapping to the replacement version.
-dict for {legacy_ver replace_ver} ${python.legacy_versions} {
-    set variant_line {variant python${legacy_ver} description }
-    append variant_line " {Legacy variant for python${legacy_ver} mapping to python${replace_ver}}"
-    append variant_line " requires python${replace_ver}"
-
-    foreach over ${python.versions} {
-        if {${legacy_ver} == ${over} || ${replace_ver} == ${over}} {
-            continue
-        }
-
-        append variant_line " conflicts python${over}"
-    }
-
-    append variant_line " { }"
-    eval $variant_line
-}
-
```

Comment:
There's a plan to add the py37 variant to mpv [1], so it's better to keep this block IMO.

[1] https://trac.macports.org/ticket/57344

Reply:
@chrstphrchvz Perhaps you should put it back?

Reply:
Alright, I will leave it, but I will also leave alone the above statement for `python.legacyversions`, since I'm not clear on what exactly the proposed changes in that ticket intend to do. I'll add comment on the ticket explaining.